pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "zpetnovazebnik"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
Do you want to keep the placeholder?

Reply:
Oh, didn't notice that. Guess it should be `PyLadies CZ`.